### DISCUSSION TITLE
PCB Order Questions

### DISCUSSION THREAD

**Discussion by Zeftronix** [cardonabits/haxo-hw#42] [Q&A] [answered]
Hi, I'm going to try building this and I had a couple of queries before I order the PCB.  I also am not going to get a semi assembled board but rather a completely unpopulated board and will be manually soldering all the smd parts (Yes, including the QFN-16 package!).

1. Is it ok to select ENIG finish with untented?  I see that you have tented listed in your ordering info and wanted to know if this is recommended or simply a matter of preference.
2. In the BOM, C2 is linked to LCSC part C28323 which is a 1uF capacitor, however the comment in your file shows C2 as 0.1uF.  Can you confirm the value of this part?
3. I assume JLCPCB will only accept this board as 3 designs even though you've panelised it yourself?  So selecting 1 design and single pcb won't work? (Just checking as it reduces cost but probably won't be accepted?).
4. And lastly, just checking that the gerbers listed here "builds/haxophone001-1.0.zip" are up to date and should result in a successful project if built correctly?

Thanks so much for your assistance.

**Comment by jcard0na** [comment, marked as answer]
Hi @Zefbot01,

Sounds like you are up for some fun!  I don't have the skills or equipment to solder the smaller components, so I cannot share my experience.  Regarding your questions:

1. I thought tented would be better to minimize possible leakage currents.  In my earlier prototypes where I sloppily soldered components and left a lot of flux around, I found that the design was sensitive to those.  But I don't think that via tenting a strict requirement.  Just for curiosity, why would you want to leave vias untented?  For testing?

2. Ah, that's an error on my part.  The audio IC datasheet calls for a 10uF and a 0.1uF cap, but the latter is optional.  I've been building haxophones with the larger cap with not problems, but yes, I'll fix that in the schematic.  If you want to follow the datasheet to the letter you could either not populate that part or use C1525.

3. That is correct.  If you choose 1 design you are given a lower price at checkout time, but later JLPC will contact you to ask you to pay for the difference.

4. Those gerbers have been used with success by several people, including me, so that would be the lowest risk option.  But if you are comfortable generating gerbers from Kicad, you might want to do that to pick up this change a28321d64bd03642627a374bd779182410e1912a, as it eliminates an annoying popping sound on startup (see https://github.com/cardonabits/haxo-rs/issues/17).

Thanks for sharing your plans and reporting the bug.  I'm looking forward to see your finished haxophone!

**Reply by jcard0na** [reply]
Cap part change commit: 83f5246d0f7a92fb96a4dcdced43ce8ae4c35762

**Reply by Zeftronix** [reply]
Thank you so much for your quick response.

1) With regard to tenting I just like the look of more of the ENIG finish so was contemplating untented but I'll probably go with tented.

2) Thanks for confirming.  The new part you linked is sized 0402.  Is it ok to order a larger 0603 package for ease of soldering?  (0804 wouldn't fit I'm guessing?).

3) Again, thanks for confirming.

4) I'm not experienced with pcb design and so a little unsure of generating correct gerbers but obviously it would be nice to have a board that eliminates the startup popping sound.  The files in the "Build" folder for JLCPCB ordering I think are out of date. Is an updated gerber zip file something you could upload by any chance?  If not, I'll try to work it out myself.

Thank you so much for your assistance.  This is a really exciting project and I'm very much looking forward to having a crack at it.

**Reply by jcard0na** [reply]
1. I suppose that you could go with untented if you apply conformal coating and/or keep your board reasonably clean.  You don't need to sacrifice aesthetics, if you like the untented look!
2. Oh boy, you are right.  That will teach me to not fixing things in a rush.  The C2 footprint is 0603, so you would need an 0603 cap.  This is it: [C14663](https://jlcpcb.com/partdetail/Yageo-CC0603KRX7R9BB104/C14663).  Incidentally I just noticed that the part I had before (C28323) was not just the wrong value but also the wrong size, 0805.  The tolerances on the 0603 footprint are large enough that it can accommodate the larger cap, that's why I did not run into production issues.  
3. 👍
4. Yes, I don't regenerate build files for every commit.  I only intend to check-in new builds after successful manufacture of a build. At the moment, I'm building a batch of haxophones for the Crowdsupply campaign.  Once all those orders have been fullfilled, I'll check in the build files I used.  That said, if your goal is to learn about PCB manufacturing, I encourage you to generate your own.  [It is not that hard!](https://www.youtube.com/watch?v=C_UdqlUmDNA)

**Reply by Zeftronix** [reply]
Well I'm about to go ahead and order some pcbs.  I generated new ones as per your suggestion but I'm a little nervous in case I've done something wrong.  Please let me know if you're imminently going to upload updated build files as I'd rather default to them if possible.

**Reply by jcard0na** [reply]
Hi @Zefbot01 

I [just pushed](https://github.com/cardonabits/haxo-hw/commit/cbc86d053a04954df30d1ef84d47345af1d40977) the build files I used for the batch build.  **Please note, that the BOM does not have [the C2 fix you found](https://github.com/cardonabits/haxo-hw/commit/d291c772256ae11b0d9371a21243f7aaba4a52ff)**. You can edit the file yourself to make that change.

Let me know how the build goes!

Cheers,

**Comment by Zeftronix** [comment]
Oh, thank you so much. I really appreciate that.
